tests/phpunit/payment-methods/class-wc-stripe-upe-payment-method-test.php

```
@@ -796,6 +796,62 @@ public function test_non_card_methods_are_not_available_when_optimized_checkout_
+		/** @var WC_Stripe_UPE_Payment_Method $mocked_payment_method */
+		$mocked_payment_method = $this->getMockBuilder( $payment_method_class )
+			->onlyMethods( $mocked_methods )
+			->getMock();
+
+		$mocked_payment_method->method( 'get_capabilities_response' )->willReturn( 'active' );
+		$mocked_payment_method->method( 'get_woocommerce_currency' )->willReturn( $currency );
+		$mocked_payment_method->method( 'is_subscription_item_in_cart' )->willReturn( false );
+
+		$this->assertFalse( $mocked_payment_method->supports_deferred_intent() );
+		$this->assertTrue( $mocked_payment_method->is_available() );
+	}
```

PHPMD (2.15.0)

[error] The method test_non_deferred_intent_methods_are_available_in_oc_mode is not named in camelCase. (undefined)

(CamelCaseMethodName)